.evergreen/config.yml

```diff
@@ -1174,40 +1204,22 @@ task_groups:
       - ".serverless"
 
 buildvariants:
-# there are no 5.1 toolchains for Ubuntu 20.04. unfortunately as of now we cannot remove
-# entire buildvariants via rule from the main matrix so we need to split this up. see EVG-13092
-- matrix_name: "tests-5.1"
+
+- matrix_name: "min-version-compile"
   matrix_spec:
+  # Ubuntu 20.04 does not have Swift 5.1 toolchains.
     os-fully-featured:
-      - "ubuntu-18.04"
       - "macos-10.15"
+      - "ubuntu-18.04"
     swift-version: "5.1"
-    ssl-auth: "*"
-  display_name: "${swift-version} ${os-fully-featured} ${ssl-auth}"
+  display_name: "${swift-version} ${os-fully-featured}"
```

Comment:
can we add "Compile" to the beginning of this matrix's name to differentiate it from the test running matricies?